icu4j/main/classes/core/src/com/ibm/icu/impl/PatternProps.java

```diff
@@ -119,6 +119,29 @@ public static String trimWhiteSpace(String s) {
         return s.substring(start, limit);
     }
 
```

Comment:
trimSpaceChar is a tweaked copy of trimWhiteSpace - formatted with clang-format rather than keeping consistent formatting.